### ISSUE TITLE
Error while parsing some emails in github.com/sendgrid/go-gmime/blob/master/gmime/fixtures/

### ISSUE THREAD

**Issue by azumakuniyuki** [azumakuniyuki/p5-Sisimai#62]
https://github.com/sendgrid/go-gmime/blob/master/gmime/fixtures/DSN-bounce.eml
```
 ***warning: Strange date format [2014-03-26 00-01-19] at lib/Sisimai/Time.pm line 411.
Use of uninitialized value $datestring in pattern match (m//) at lib/Sisimai/Data.pm line 222.
```

**Comment by azumakuniyuki** [comment]
* https://github.com/sendgrid/go-gmime/blob/master/gmime/fixtures/DSN-bounce.eml
* https://github.com/sendgrid/go-gmime/blob/master/gmime/fixtures/NDR-block.eml

**Comment by azumakuniyuki** [comment]
Fixed code at commit 8b81d1f3677dd6a4b5364c2dd713a62f483172fa
